app/src/ui/changes/changes-list.tsx

```diff
@@ -31,6 +31,7 @@ import { IAutocompletionProvider } from '../autocompletion'
 import { showContextualMenu } from '../main-process-proxy'
 import { arrayEquals } from '../../lib/equality'
 import { clipboard } from 'electron'
+import * as path from 'path'
```

Comment:
:art: instead of importing everything, why not just `import { parse } from 'path'`?

Our production minification steps should tidy this up anyway, but it'll simplify the change below and align better with how we import specific functions from core modules.